modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -57,6 +61,51 @@ async function getSiteHomePageScreenshot(
 	return screenshot;
 }
 
+test('can XSS with `searchContainerId` in Asset Libraries import', async ({
+	apiHelpers,
+	depotAdminPage,
+	page,
+}) => {
+	const depotName = getRandomString();
+
+	await apiHelpers.jsonWebServicesDepot.addDepotEntry(depotName);
+
+	await depotAdminPage.goto();
+
+	await depotAdminPage.page.getByRole('link', {name: depotName}).click();
```

Comment:
maybe we can define it in the depotAdminPage as well, as goToDepotByName or something. Just in case in the future the 'link' type changes into something else (it maybe won't but I have seen a lot of changes recently in the UI :P) But I think you're more like an expert here than I am, so if you think it is somehow redundant, leave it as it is.